src/js/tracker.js

```diff
@@ -2666,7 +2641,34 @@
 			preservePageViewId: function () {
 				preservePageViewId = true
 			}
-		};
+	};
+
+	if ('__FORM_TRACKING_ENABLED__' === 'true') {
+			var formTrackingManager = require('./forms').getFormTrackingManager(core, trackerId, addCommonContexts);
+			/**
+			 * Enables automatic form tracking.
+			 * An event will be fired when a form field is changed or a form submitted.
+			 * This can be called multiple times: only forms not already tracked will be tracked.
+			 *
+			 * @param object config Configuration object determining which forms and fields to track.
+			 *                      Has two properties: "forms" and "fields"
+			 * @param array context Context for all form tracking events
+			 */
+			API.enableFormTracking = function (config, context) {
+					if (mutSnowplowState.hasLoaded) {
+							formTrackingManager.configureFormTracking(config);
+							formTrackingManager.addFormListeners(context);
+					} else {
+							mutSnowplowState.registeredOnLoadHandlers.push(function () {
+									formTrackingManager.configureFormTracking(config);
+									formTrackingManager.addFormListeners(context);
+							});
+					}
+			};
+		}
+
+		return API;
+
```

Comment:
The indent seems to be off.

Also does this replace #677 ? If so you can close out the other PR :+1: